src/server/config.rs

```diff
@@ -91,4 +120,17 @@ mod tests {
         cfg.raft_store.raft_heartbeat_ticks = 0;
         assert!(cfg.validate().is_err());
     }
+
+    #[test]
+    fn test_end_point_txn_concurrency() {
```

Comment:
Add another case to check line 105, where `end_point_txn_concurrency_on_busy == 0`.